### PR TITLE
RES-83 Add NAB to NuPIC regression tests

### PR DESCRIPTION
**Pull request by SaganBolliger**
https://jira.numenta.com/browse/RES-83

Under development. Don't merge.

### CODE REVIEW

File: tests/anomaly/NAB/benchmark_results.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "numenta": {
```
Comment (BoltzmannBrain) [inline tests/anomaly/NAB/benchmark_results.json]: Use the results currently checked in [here](https://github.com/numenta/NAB/blob/master/results/final_results.json#L7).

Comment (SaganBolliger) [inline tests/anomaly/NAB/benchmark_results.json]: @BoltzmannBrain Ok, I've updated the values.

File: .travis.yml [hunk 2]
```diff
@@ -47,6 +47,9 @@ before_install:
   - (cd nupic && git reset --hard ${NUPIC_SHA})
   - NUPIC_CORE_VERSION=`python get_nupic_bindings_version.py`
 
+  # Clone NAB
+  - git clone https://github.com/numenta/NAB.git
```
Comment (rhyolight) [inline .travis.yml]: Add `--depth 50` to this command to decrease the size of the checkout.

**Comment by rhyolight** [comment]
@SaganBolliger From the [Travis-CI docs](http://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables):

> Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code.

That is why your build is failing on this PR. I recommend you merge it and see what happens on `master`.

**Comment by SaganBolliger** [comment]
@rhyolight Can you merge this so we can test it on master?

**Comment by SaganBolliger** [comment]
@rhyolight Seems like there is an issue with this. 
https://travis-ci.org/numenta/nupic.regression/builds/85402469
Can you reopen the PR?

**Comment by rhyolight** [comment]
@SaganBolliger You can't re-open a PR once it has been merged. You need to create a new one. See my [note here](https://github.com/numenta/nupic.regression/commit/ae2744a394a5732b511c6bfcdce5af48131aa882#commitcomment-13776828).

File: tests/anomaly/NAB/benchmark_results.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+        "reward_low_FN_rate": 69.28308927558334,
+        "reward_low_FP_rate": 56.50002521814655,
+        "standard": 64.70049598232758
+    }
+}
```
Comment (scottpurdy) [inline tests/anomaly/NAB/benchmark_results.json]: missing newline at end of file

**Comment by scottpurdy** [comment]
Please change file/directory names to all lowercase

**Pull request merged**